nextflow_schema.json

```diff
@@ -742,10 +742,10 @@
                 },
                 "vep_cache_version": {
                     "type": "integer",
-                    "default": 110,
+                    "default": 112,
                     "description": "Specify the version of the VEP cache provided to the `--vep_cache` option.",
                     "fa_icon": "fas fa-align-center",
-                    "enum": [107, 110]
+                    "enum": [107, 110, 112]
```

Comment:
Should we allow 111 as well?

Reply:
Kind of thinking we should remove this enum altogether and just have the requirement on it being a number.

Reply:
Yeah, that's better actually 😅